nixos/modules/virtualisation/cri-o.nix

```diff
@@ -126,7 +126,7 @@ in
       documentation = [ "https://github.com/cri-o/cri-o" ];
       wantedBy = [ "multi-user.target" ];
       after = [ "network.target" ];
-      path = [ cfg.package ];
+      path = [ cfg.package ] ++ optional (cfg.storageDriver == "zfs") config.boot.zfs.package;
```

Comment:
Please change it to how podman does this.

https://github.com/NixOS/nixpkgs/blob/4d628f4b73a6ed8dece33b40c8704d8164dd252d/nixos/modules/virtualisation/podman/default.nix#L8-L11

Reply:
Sorry, I meant for it to be moved `crioPackage` as well.

Reply:
@SuperSandro2000 Do you want to do the `crioPackage` change and preserve your commit signing or should I do it?

Reply:
applied your diff